src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -286,17 +286,22 @@ public function serialize() {
 		$data            = $this->fetch_data();
 		$data['post_id'] = $this->post_id;
 
-		return serialize( $data );
+		return wp_json_encode( $data );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: be6b3b73926fdef1dfd04fd37b1b8f04</sub>